examples/favorites_advanced/favorites_advanced_base/lib/src/repositories/puppies_repository.dart

```diff
@@ -94,8 +94,44 @@ class PuppiesRepository {
     return puppiesWithExtraData;
   }
 
+  // Future<Puppy> fetchPuppyExtraDetails({required Puppy puppy}) async {
```

Comment:
@StanislavPrime, if you don't need this code could you please remove it ?

Reply:
Yes, this code was removed.